src/gax/src/options/mod.rs

```diff
@@ -48,6 +49,22 @@ pub struct RequestOptions {
 }
 
 impl RequestOptions {
+    /// Treat the RPC underlying this method as idempotent.
+    pub fn set_idempotent(&mut self) {
```

Comment:
Should this and the following RPC be combined and just accept a bool? Also, should we mention what setting this means for the request, i.e. how it relates retrying?

Reply:
Done.